Havit/Havit/Screens/Category/View/CategoryViewController.swift

```diff
@@ -7,23 +7,184 @@
 
 import UIKit
 
+import SnapKit
+
 class CategoryViewController: UIViewController {
 
+    // MARK: - Vars & Lets Part
+
+    weak var coordinator: CategoryCoordinator?
+
+    // MARK: - UI Component Part
+
+    private lazy var categoryCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.register(cell: CategoryCollectionViewCell.self)
+
+        return collectionView
+    }()
+
+    private let categoryCountLabel: UILabel = {
+        let label = UILabel()
+        label.font = .font(FontName.pretendardReular, ofSize: 13)
+        label.text = "전체 0"
+        label.textColor = UIColor.gray003
+
+        return label
+    }()
+
+    // 📌 카테고리 추가 버튼은 재사용될 것 같아서 나중에 따로 빼면 좋을 것 같아요 !
+    private let addButton: UIButton = {
+        var configuration = UIButton.Configuration.plain()
+
+        var container = AttributeContainer()
+        container.font = .font(FontName.pretendardSemibold, ofSize: 12)
+
+        configuration.attributedTitle = AttributedString("카테고리 추가", attributes: container)
+
+        configuration.baseForegroundColor = UIColor.purpleText
+        configuration.image = UIImage(named: "category_add")
+
+        configuration.background.cornerRadius = 23
+        configuration.background.strokeColor = UIColor.purpleLight
+        configuration.background.strokeWidth = 1
+
+        configuration.imagePadding = 2
+        configuration.contentInsets = NSDirectionalEdgeInsets(top: 2, leading: 10, bottom: 3, trailing: 10)
+        configuration.imagePlacement = .leading
+
+        let button = UIButton(configuration: configuration, primaryAction: nil)
+
+        return button
+    }()
+
+    // NavigationBar Button
+    private let backButton: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: "iconBackBlack"), for: .normal)
+        return button
+    }()
+
+    private let editButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("수정", for: .normal)
+        button.titleLabel?.font = .font(FontName.pretendardMedium, ofSize: 14)
+        button.setTitleColor(UIColor.gray003, for: .normal)
+        return button
+    }()
+
+    // MARK: - Life Cycle Part
     override func viewDidLoad() {
         super.viewDidLoad()
+        setDelegation()
+        setLayouts()
+        setNavigationBar()
+    }
+
+    // MARK: - Custom Method Part
+
+    private func setDelegation() {
+        categoryCollectionView.delegate = self
+        categoryCollectionView.dataSource = self
+    }
+
+    private func setNavigationBar() {
+        title = "전체 카테고리"
+        // 여기 폰트적용 어떻게 하나용
+        navigationController?.navigationBar.titleTextAttributes = [.font: UIFont(name: "Pretendard-Bold", size: 16)]
```

Comment:
p3;
요렇게도 사용할 수 있습니다
```suggestion
navigationController?.navigationBar.titleTextAttributes = [.font: UIFont.font(.pretendardBold, ofSize: 16)]
```
그나저나 저 코드 안먹히나요?

Reply:
@sujinnaljin 안먹히지는 않는데 경고가 떠서 ..!! 여쭤봤습니다 .. 제안해주신 코드 반영했더니 사라졌어요 ~~  최고최고